### ISSUE TITLE
Register package

### ISSUE THREAD

**Issue by matbesancon** [jump-dev/MatrixOptInterface.jl#12]
I think there could be some cleanup and then registering the package. Depending on still requires manual cloning for now

**Comment by blegat** [comment]
It's not ready for release yet, we should at least make sure there is a warning for the code path given in https://github.com/jump-dev/MatrixOptInterface.jl/issues/6. The LP/conic readers/writers could be improved too, so if we release, it would be a v0.0.1 with a warning that things might break at v0.0.2 ^^ I'd like to find some time to clean up this package for release soon but haven't found yet.

Here are some breaking changes blocking a release

- [x] https://github.com/jump-dev/MatrixOptInterface.jl/pull/17
- [ ] https://github.com/jump-dev/MatrixOptInterface.jl/issues/19
- [ ] https://github.com/jump-dev/MatrixOptInterface.jl/issues/20
- [x] https://github.com/jump-dev/MatrixOptInterface.jl/pull/21
- [x] https://github.com/jump-dev/MatrixOptInterface.jl/pull/22

**Comment by blegat** [comment]
An idea to merge part of this into MOI is outlined in https://github.com/jump-dev/MathOptInterface.jl/issues/1261.
This packages could then be easy constructors to build the right `MOIU.GenericModel` with the right type arguments and the matrices are the right places but not actually defining many new `MOI.ModelLike`.

**Comment by matbesancon** [comment]
@blegat we should really register it, it's a bother to depend on (see the hacky PR on DiffOpt to make the build work on CI with the commit version which does not depend on MOI 0.10)

**Comment by matbesancon** [comment]
given the months in which it was not breaking, this could definitely have been a 0.1 release

**Comment by blegat** [comment]
Now that JuMP v0.22 is released, I'm preparing a PR to DiffOpt so that it uses MOI.Utilities.MatrixOfConstraints and does not need this package.

**Comment by matbesancon** [comment]
ok that also works, can you pick up the one I started on DiffOpt?

**Comment by blegat** [comment]
Yes, I continued based on your PR, you can merge it into your branch: https://github.com/jump-dev/DiffOpt.jl/pull/151
